components/user/UserProfile.js

```diff
@@ -68,9 +77,16 @@ function UserProfile({ BASE_URL, data }) {
 
       <div className="flex justify-center my-4">
         {qrShow && (
-          <QRCodeSVG value={`${BASE_URL}/${data.username}`} size={fallbackImageSize * 2} />
+          <QRCodeCanvas
+            value={`${BASE_URL}/${data.username}`}
+            size={fallbackImageSize * 2}
+            id="qrcode"
+          />
         )}
       </div>
+      <div className="flex justify-center mb-4" >
+        {qrShow && (<Button text="Download QR code" primary={true} onClick={downloadQR} />)}
+      </div>
```

Comment:
Formatting is not consistent with the formatting changes on `71`

```suggestion
      <div className="flex justify-center mb-4">
        {qrShow && (
          <Button text="Download QR code" primary={true} onClick={downloadQR} />
        )}
      </div>
```